guidance_pipeline/main.go

```diff
@@ -52,7 +133,7 @@ func makeMapOfRegs(header string, records [][]string) map[string][]Guidance {
 			for _, reg := range regs {
 				guidance := Guidance{
 					Name: record[0],
-					Link: record[1],
+					Link: validURL(header, record[1]),
```

Comment:
To prevent the user seeing broken links this should return an error and not get added to `guidance` if it is an invalid url.